index.js

```diff
@@ -36,24 +36,71 @@ function getCartTotal(cart) {
   - Any of the products in the `products` array does not have a `priceInCents` key.
 */
 function filterProductsByPriceRange(products, min, max) {
-  const result = [];
-  for (let product of products) {
-    if (product.priceInCents >= min && product.priceInCents <= max) {
-      result.push(product);
+  if ((products.length == 0) || (products.some(x => (x.priceInCents === undefined)))) {
+    throw "error1"
```

Comment:
in the future you should include a message with what exactly is going wrong. It'll help you debug as a developer or let the user know what information is incorrect/missing